pkg/authorization/registry/rolebinding/virtual_registry.go

```diff
@@ -110,12 +112,14 @@ func (m *VirtualRegistry) CreateRoleBinding(ctx kapi.Context, roleBinding *autho
 	return nil
 }
 
-func (m *VirtualRegistry) UpdateRoleBinding(ctx kapi.Context, roleBinding *authorizationapi.RoleBinding) error {
+func (m *VirtualRegistry) UpdateRoleBinding(ctx kapi.Context, roleBinding *authorizationapi.RoleBinding, skipEscalationCheck bool) error {
```

Comment:
!skipEscalationCheck is hurting my brain... can we make the check `!allowEscalation`?